virttest/utils_misc.py

```diff
@@ -3389,33 +3389,45 @@ class struct_timespec(ctypes.Structure):
         return monotonic_time_os()
 
 
-def verify_host_dmesg(dmesg_log_file=None, ignore_result=False, level_check=3):
+def verify_dmesg(dmesg_log_file=None, ignore_result=False, level_check=3,
+                 session=None):
```

Comment:
Seems relatively straight forward, so OK. I was going to suggest that instead of a session and handle things differently, that a runner callable was passed, defaulting to `process.run`, plus **runner_kwargs, but that's not that big a deal.